kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/cluster/Kubernetes.java

```diff
@@ -54,4 +54,14 @@ public KubeCmdClient defaultCmdClient() {
     public String toString() {
         return CMD;
     }
+
+    public boolean isOpenshift() {
+        try {
+            return Exec.exec(CMD, "api-versions").out().contains("openshift.io");
```

Comment:
You need to check the exit status from the programs.  Otherwise we end up with silently failing code.